server/boot/authentication.js

```diff
@@ -0,0 +1,4 @@
+module.exports = function enableAuthentication(server) {
+  // enable authentication
```

Comment:
LB boilerplate, right? The comment is text book "don't write comments like these"... it's just the method name, converted from "camelcase" to "space separated" (with acronym expanded) :-(